src/Command/AutoMergeCommand.php

```diff
@@ -124,10 +124,10 @@ private function mergeBranches(Package $package, array $projectConfig): void
                 }
             } catch (RuntimeException $e) {
                 if (409 === $e->getCode()) {
-                    $message = sprintf('Merging of %s into %s contains conflicts. Skipped.', $head, $base);
+                    $message = sprintf('%s: Merging of %s into %s contains conflicts. Skipped.', $repositoryName, $head, $base);
```

Comment:
I think we can avoid the `sprintf` entirely with placeholders, let me check

Reply:
Nevermind, the replacements I found in `LineFormatter` are for the format string, not the message.